tests/bluetooth/controller/common/include/helper_features.h

```diff
@@ -270,3 +270,4 @@ FEAT_LE_PATH_LOSS \
  * 8 Bytes for feature exchange
  */
 #define EXPECTED_FEAT_EXCH_VALID 0x0000000FF787CF2F
```

Comment:
These 'magic' numbers - why are they what they are?

Reply:
They are from the Core Spec V 5.2, Volume 6, Part B, 4.6: The column named "Valid from Controller to Controller". But yeah maybe add a comment for that

Reply:
Agreed; I'll add a reference; same goes for the definition in ll_feat.h